src/renderers/shared/fiber/ReactFiberScheduler.js

```diff
@@ -285,12 +289,23 @@ module.exports = function<T, P, I, TI, C, CX>(config : HostConfig<T, P, I, TI, C
     }
   }
 
-  function commitAllLifeCycles() {
+  function commitAllLifeCyclesAndApplyInitialEffects() {
```

Comment:
Let's keep the previous name. What you're calling an "initial effect" could also be used to describe `componentDidMount`. I think calling this a "lifecycle" is sufficient.